tequilapi/endpoints/sse.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package endpoints
+
+import (
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+// SSEHandler represents the sse handler
+type SSEHandler interface {
+	Sub(resp http.ResponseWriter, req *http.Request, params httprouter.Params)
+}
+
+// AddRoutesForSSE adds route for sse
+func AddRoutesForSSE(router *httprouter.Router, handler SSEHandler) {
+	router.GET("/sse", handler.Sub)
```

Comment:
SSE is content type, I would prefer more descriptive url like "/events", "/service-states" or something similar

Reply:
renamed to /events